api/src/main/java/org/pmiops/workbench/cohortbuilder/CohortQueryBuilder.java

```diff
@@ -81,25 +36,12 @@ public class CohortQueryBuilder {
 
   private static final String UNION_TEMPLATE = "UNION ALL\n";
 
-  private static final String INCLUDE_SQL_TEMPLATE = "${mainTable}.person_id IN (${includeSql})\n";
-
-  private static final String PERSON_ID_WHITELIST_PARAM = "person_id_whitelist";
-  private static final String PERSON_ID_BLACKLIST_PARAM = "person_id_blacklist";
-
-  private static final String PERSON_ID_WHITELIST_TEMPLATE =
-      "${mainTable}.person_id IN unnest(@" + PERSON_ID_WHITELIST_PARAM + ")\n";
-  private static final String PERSON_ID_BLACKLIST_TEMPLATE =
-      "${mainTable}.person_id NOT IN unnest(@" + PERSON_ID_BLACKLIST_PARAM + ")\n";
-
-  private static final String EXCLUDE_SQL_TEMPLATE =
-      "${mainTable}.person_id NOT IN\n" + "(${excludeSql})\n";
-
-  private CBCriteriaDao cbCriteriaDao;
+  //  private CBCriteriaDao cbCriteriaDao;
   private static final Logger log = Logger.getLogger(CohortQueryBuilder.class.getName());
 
   @Autowired
-  public CohortQueryBuilder(CBCriteriaDao cbCriteriaDao) {
-    this.cbCriteriaDao = cbCriteriaDao;
+  public CohortQueryBuilder() {
+    // this.cbCriteriaDao = cbCriteriaDao;
```

Comment:
I'm assuming this line should be removed?

Reply:
Yes. cbCriteriaDao is not used